admin-core/build.gradle.kts

```diff
@@ -122,24 +122,41 @@ application {
     applicationDefaultJvmArgs = defaultJvmArgs + devJvmArgs
 }
 
-val jibExtraDirs = "$buildDir/jib-extra-dirs"
+val registryName = "ghcr.io"
+val fullImageTag = "$registryName/drill4j/admin"
+val apiPort = "8090"
+val debugPort = "5006"
+val secureApiPort = "8453"
+val gitUsername = System.getenv("GH_USERNAME") ?: ""
+val gitPassword = System.getenv("GH_TOKEN") ?: ""
 jib {
     from {
         image = "adoptopenjdk/openjdk11:latest"
     }
     to {
-        image = "ghcr.io/drill4j/admin"
+        image = fullImageTag
         tags = setOf(version.toString())
+        auth {
+            username=gitUsername
+            password=gitPassword
+        }
     }
     container {
-        ports = listOf("8090", "5006", "8453")
-        volumes = listOf("/distr", "/work")
+        ports = listOf(apiPort, debugPort , secureApiPort)
+        volumes = listOf("/config", "/config/ssl")
```

Comment:
I would like us to have a _single_ volume - to reduce complexity
E.g. in Kubernetes creating volumes requires 3 steps, 1 config for each-> 2 volumes is x2 config lines

Everything can be placed in one volume:
```
- application.conf
- somesslcert.jks
- something-else.foo
```